### DISCUSSION TITLE
Would it be possible to provide support for the Nucleo F446ZE card?

### DISCUSSION THREAD

**Discussion by 7heS4ge** [stm32duino/Arduino_Core_STM32#1970] [Variants] [answered]
Would it be possible to provide support for the Nucleo F446ZE card?
Or can I choose another board in Arduino IDE (like F207ZG) ?

My problem is I attempt to use the Adafruit RG Matrix Panel library, but I have compilation error because the library check this definitions : ARDUINO_ARCH_ESP32, __AVR_ATmega2560__
What definitions can I add to the code to make it work ? Is Nucleo F446ZE like an Arduino UNO, or like a Arduino MEGA, AVR microprocessor ?

**Comment by fpistm** [comment]
Hi @7heS4ge 
The definition to use is: `ARDUINO_ARCH_STM32`

About Nucleo F446ZE, I've added the request to the #722 
You could at least add the generic support:
https://github.com/stm32duino/Arduino_Core_STM32/wiki/Add-a-new-variant-%28board%29

**Reply by fpistm** [reply, marked as answer]
Provided by #2108
